VideoAPI/Testing.Common/Helper/ApiUriFactory.cs

```diff
@@ -99,6 +99,9 @@ public static class EPEndpoints
         {
             private const string ApiRoot = "conferences";
             public static string GetEndpointsForConference(Guid conferenceId) => $"{ApiRoot}/{conferenceId}/endpoints";
+
+
+            public static string UpdateEndpointForConference(Guid conferenceId, Guid endpointId) => $"{ApiRoot}/{conferenceId}/endpoints/{endpointId}/displayname";
```

Comment:
Think we should rename to update displayname. Thanks

Reply:
renamed to UpdateDisplayNameForEndpoint